### ISSUE TITLE
Content Switching Vservers not supported

### ISSUE THREAD

**Issue by chandanjain999** [rokett/Citrix-NetScaler-Exporter#10]
First of all, thank you for the operational exporter that could even get me to this issue. 

Is Content Switching Vservers going to be added with the already present Load Balancing Vservers?
Would like to see the metrics on those Vservers for sure and trying on my own to see if I can solve this issue. Will update for any progress

**Comment by rokett** [comment]
no problem.....I'm happy it's helping people out.

I haven't had a need to add Content Switching Vservers as we don't currently use them in my org.  I'm happy to accept a pull request if you're familiar with Go?  Failing that it shouldn't be a huge amount of effort to add them so I can look at it, but probably not until some time next week.

**Comment by chandanjain999** [comment]
Ahhh... makes sense. Content Switching Vservers are vital to my org, and it's interesting to see what can be done with that data.

I'm not at all familiar with Go, in fact pretty new in the programming world, so I'll try working with it asap. If you could add that feature it would more than appreciated, but in the mean time I'll see if I can make the addition successfully myself.

**Comment by chandanjain999** [comment]
I got it all working! Nonetheless, thanks for being responsive. Great exporter

**Comment by rokett** [comment]
That's good news :-) 

I'm sure that addition would be useful in the future...are you able to raise a pull request so I can merge it?

**Comment by chandanjain999** [comment]
I can't pull with current permissions

**Comment by rokett** [comment]
I'm not sure I follow.....I've had changes merged into projects owned by others before with no specific permissions on their repo.

**Comment by chandanjain999** [comment]
$ git status
On branch cj
nothing to commit, working tree clean
$ git push origin cj
remote: Permission to rokett/Citrix-NetScaler-Exporter.git denied to chandanjain999.
fatal: unable to access 'https://github.com/rokett/Citrix-NetScaler-Exporter.git/': The requested URL returned error: 403
$
